v2.0/information-schema.md

```diff
@@ -50,6 +50,22 @@ Column | Description
 `NUMERIC_SCALE` | If `DATA_TYPE` is an exact numeric type, the scale (i.e. number of digits to the right of the decimal point); otherwise *NULL*.
 `DATETIME_PRECISION` | Always *NULL* (unsupported by CockroachDB).
 
+### column_privileges
+
+The `column_privileges` view identifies all privileges granted on columns to a currently enabled role or by a currently enabled role.
+There is one row for each combination of grantor, grantee, and column (defined by table_catalog, table_schema, table_name, and column_name).
```

Comment:
nit: wrap the column names in backticks.

Reply:
Done.